ush/python/pygfs/jedi/jedi.py

```diff
@@ -216,18 +218,51 @@ def get_bias_dict(self, task_config: AttrDict) -> Dict[str, Any]:
                 obfile = ob['obs bias']['input file']
                 obdir = os.path.dirname(obfile)
                 basename = os.path.basename(obfile)
-                prefix = '.'.join(basename.split('.')[:-2])
-                for file in ['satbias.nc', 'satbias_cov.nc', 'tlapse.txt']:
-                    bfile = f"{prefix}.{file}"
-                    copylist.append([os.path.join(task_config.COM_ATMOS_ANALYSIS_PREV, bfile), os.path.join(obdir, bfile)])
-                    # TODO: Why is this specific to ATMOS?
+                prefix = '.'.join(basename.split('.')[:-3])
+                bfile = f"{prefix}.{bias_file}"
+                tar_file = os.path.join(obdir, bfile)
+                copylist.append([os.path.join(task_config.VarBcDir, bfile), tar_file])
+                break
 
         bias_dict = {
             'mkdir': [os.path.join(task_config.DATA, 'bc')],
             'copy': copylist
         }
+
         return bias_dict
 
+    @logit(logger)
+    def extract_tar(self, tar_file: str) -> None:
+        """Extract bias correction files from a tarball
+
+        This method extract files from a tarball
+
+        Parameters
+        ----------
+        tar_file
+            path/name of tarball
+
+        Returns
+        ----------
+        None
+        """
+
+        # extract files from tar file
+        tar_path = os.path.dirname(tar_file)
+        try:
+            with tarfile.open(tar_file, "r") as tarball:
+                tarball.extractall(path=tar_path)
+                logger.info(f"Extract {tarball.getnames()}")
+        except tarfile.ReadError as err:
+            if tarfile.is_tarfile(tarfile[1]):
```

Comment:
```suggestion
            if tarfile.is_tarfile(tar_file):
```
`tar_file` is the name of the file. `tarfile` is the python module.

Reply:
Sloppy oversight on my part.  Thank you @aerorahul for catching my mistake.  `tarfile[1]` replaced by `tar_file`.  Done at 821ee760.